src/deepsparse/loggers/base_logger.py

```diff
@@ -32,12 +32,7 @@ def log(self, identifier: str, value: Any, category: Optional[str] = None):
         and then possibly process the information and pass it to
         the monitoring service
 
-        :param identifier: The identifier of the log
-             By default should consist of at least one string (name of the pipeline):
-                e.g. identifier = "pipeline_name"
-            If identifier is to be more complex, optional strings are to be concatenated
-            using dot as a separator:
-                e.g. identifier = "pipeline_name.some_argument_1.some_argument_2"
+        :param identifier: The name of the thing that is being logged.
```

Comment:
Why the shortened docstring? Is this not the expected format for identifier anymore?

Reply:
Not entirely. The identifier for logging **data** should by default has this structure. But this is not true for logging **system** information